### PR TITLE
<PrevieWidget/>- mobile type

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary

Related to #3833

New component: https://zpl.io/2jMDQLr

This PR is related to the `mobile` type without the custom content area size. 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

File: src/PreviewWidget/PreviewWidget.st.css [hunk 1]
```diff
@@ -22,17 +22,70 @@
...
+  transform: translate(-50%, -50%);
+}
+
+.content{
+  -st-states: type(enum(blank,mobile, browserBar));
+}
+
+.content:type(mobile){
+  height: 480px;
+  width: 320px;
```
Comment (nitayneeman) [inline src/PreviewWidget/PreviewWidget.st.css]: What does happen with smaller content? Maybe it should be `maxWidth` & `maxHeight`?

File: src/PreviewWidget/docs/index.story.js [hunk 2]
```diff
@@ -30,15 +30,14 @@ const childNode = (
...
-    backgroundColor="D80"
+    backgroundColor="Y30"
+    height="100%"
   >
     <Text>Content goes here</Text>
   </Box>
 );
 
-const childNodeString = `<Box align=\'center\' verticalAlign=\'middle\' height=\'50px\' width=\'150px\' backgroundColor=\'D80\'>\n
+const childNodeString = `<Box height=\'100%\' align=\'center\' verticalAlign=\'middle\' backgroundColor=\'Y30\'>\n
```
Comment (nitayneeman) [inline src/PreviewWidget/docs/index.story.js]: If you use backticks (``) you don't need to escape these characters or use `\n`  :)

File: src/PreviewWidget/test/PreviewWidget.visual.js [hunk 3]
```diff
@@ -7,38 +7,44 @@ import PreviewWidget from '../PreviewWidget';
 import Box from 'wix-style-react/Box';
```
Comment (nitayneeman) [inline src/PreviewWidget/test/PreviewWidget.visual.js]: Currently these tests are fine, but maybe we'll merge some of them later (after we'll have more use-cases) to reduce screenshots

File: src/Foundation/stylable/colors.st.css [hunk 4]
```diff
@@ -98,6 +98,7 @@
   D10-05: calc_rgba(value(D10), 0.05);
   D10-10: calc_rgba(value(D10), 0.1);
   D10-20: calc_rgba(value(D10), 0.2);
+  D10-24: calc_rgba(value(D10), 0.24);
```
[the remark this answers is not in the document]
Comment (TalZa1810) [reply src/Foundation/stylable/colors.st.css]: agreed with Ben to change it to D10-30.

File: src/PreviewWidget/PreviewWidget.st.css (hunk 1 above)
Comment (TalZa1810) [reply src/PreviewWidget/PreviewWidget.st.css]: Currently, we've decided to have a fixed height and width. The customizable part will be in a different PR.

**Comment by TalZa1810** [comment]
We've split the `PreviewWidget` component to 3 components. Created a different PR for the mobile type.

**Pull request closed without merging**